modules/500-upmeter/templates/status/ingress.yaml

```diff
@@ -5,7 +5,7 @@ kind: Ingress
 metadata:
   name: status
   namespace: d8-{{ .Chart.Name }}
-  {{- include "helm_lib_module_labels" (list . (dict "app" "status")) | nindent 2 }}
+  {{- include "helm_lib_module_labels" (list . (dict "app" "status" "deckhouse.io/export-domain" "status")) | nindent 2 }}
```

Comment:
Let's do the same for the upmeter domain.

Reply:
I expected it to be an annotation. Why is it a label?

Reply:
never mind. It became an annotation :)